pages/dkp/kommander/2.1/licensing/purchase/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: layout.pug
+navigationTitle: Purchase a Kommander license
+title: Purchase
+excerpt: Obtain licenses for Kommander clusters
+menuWeight: 0
```

Comment:
```suggestion
menuWeight: 20
```

Reply:
So, don't know if this is right, but purchase and remove were both set to `0`.

So, I was thinking that it should be 
Licensing
- Add a license (most important)
- Purchase a license
- Remove a license 

But, feel free to call me on this

Reply:
yeah that''s a good call Matt. Thank you!